docs/guides/jsdom.md

```diff
@@ -11,6 +11,33 @@ gets run *before* React's code is run.
 
 As a result, a standalone script like the one below is generally a good approach:
 
+`jsdom v10~`:
+
+```js
+/* setup.js */
+
+var { JSDOM } = require('jsdom');
+var jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+var { window } = jsdom;
+
+global.window = window;
+global.document = window.document;
+
+Object.keys(window).forEach((property) => {
+  if (typeof global[property] === 'undefined') {
+    global[property] = window[property];
```

Comment:
this should use `defineProperty` with a property descriptor to maintain nonenumerability and nonwritability when applicable.